tinygrad/features/graph.py

```diff
@@ -98,3 +98,12 @@ def graph_uops(uops:List[UOp]):
     G.add_node(uops.index(u), label=f"{str(u.uop)[5:]}{(' '+str(u.arg)) if u.arg is not None else ''}\n{str(u.dtype)}", style="filled", fillcolor=colors.get(u.uop, "#ffffff"))  # noqa: E501
     for v in u.vin: G.add_edge(uops.index(v), uops.index(u))
   save_graph(G, f'{GRAPHPATH}.uops', '-Grankdir=LR')
+
+SCHEDULES: List[ScheduleItem] = []
+def graph_schedule(schedule:List[ScheduleItem]):
```

Comment:
I wouldn't call this graph, it doesn't have anything to do with graph. SAVE_SCHEDULE? (and no need for the function, put the logic in schedule.py)